### PR TITLE
Feature/ Check Level Signature (Project Service)

### PR DESCRIPTION
**Pull request by wanyaland**
- Added Flag for checking claGroups signed at Foundation or Project Level

Signed-off-by: wanyaland <wanyaland@gmail.com>

### CODE REVIEW

File: cla-backend-go/project/service.go [hunk 1]
```diff
@@ -61,6 +64,19 @@ func (s service) GetCLAGroupByID(projectID string) (*models.Project, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	log.Warnf("Checking for foundationSFID: %s CLA Groups", project.FoundationSFID)
```
Comment (dealako) [inline cla-backend-go/project/service.go:68]: Debug?

File: cla-backend-go/project/service.go [hunk 2]
```diff
@@ -61,6 +64,19 @@ func (s service) GetCLAGroupByID(projectID string) (*models.Project, error) {
...
+	pcgs, pcgErr := s.projectCGRepo.GetProjectsIdsForFoundation(project.FoundationSFID)
+	if pcgErr != nil {
+		return nil, pcgErr
+	}
+
+	if signedAtFoundationLevel(pcgs) {
+		project.FoundationLevelCLA = true
+	}
+
+	log.Warnf("Got Project CLA Groups : %+v for foundation SFID: %s", pcgs, project.FoundationSFID)
```
Comment (dealako) [inline cla-backend-go/project/service.go:78]: Let's do Debugf

File: cla-backend-go/project/service.go [hunk 3]
```diff
@@ -61,6 +64,19 @@ func (s service) GetCLAGroupByID(projectID string) (*models.Project, error) {
...
+
+	log.Warnf("Checking for foundationSFID: %s CLA Groups", project.FoundationSFID)
+	pcgs, pcgErr := s.projectCGRepo.GetProjectsIdsForFoundation(project.FoundationSFID)
+	if pcgErr != nil {
+		return nil, pcgErr
+	}
+
+	if signedAtFoundationLevel(pcgs) {
+		project.FoundationLevelCLA = true
+	}
```
Comment (dealako) [inline cla-backend-go/project/service.go:76]: Default is false then, right?

Comment (wanyaland) [reply cla-backend-go/project/service.go:76]: Yes default is false. Either project level or no CLA groups association

**Pull request merged**